module/views/header_element.tpl

```diff
@@ -70,7 +82,7 @@
          <a id="hosts-states-popover"
             class="hosts-all" data-count="{{ h['nb_elts'] }}" data-problems="{{ h['nb_problems'] }}"
             href="/all?search=type:host"
-            data-original-title="Hosts states" data-toggle="popover popover-hosts" title="Overall hosts states: {{h['nb_elts']}} hosts, {{h["nb_problems"]}} problems" data-html="true" data-trigger="hover">
```

Comment:
You know, I'm not actually sure if there was a reason for the removal, might've just been something that happened as I was working on it.

I spent some time reworking parts of the HTML quite heavily before realizing it could be done with smaller increments, so it's possible these are just artefacts of partially reverting those changes.